tests/main/index.js

```diff
@@ -0,0 +1,37 @@
+describe('Main Controller', function () {
+
+    var gameService;
+
+    beforeEach(module("app"));
+
+    var ctrl;
+
+    beforeEach(function () {
+        inject(function (GameService) {
+            gameService = GameService;
+        });
+
+        inject(function ($controller) {
+            ctrl = $controller("MainController", {
+                GameService: gameService
+            });
+        });
```

Comment:
Its a small thing, but you can use `inject` just once if you want, 

    inject(function(GameService, $controller) {